### PR TITLE
Issues tests

### PR DESCRIPTION
**Pull request by manuelq12**
Github API testing creating & updating issues using POST & PATCH.

### CODE REVIEW

File: test/features/support/GithubAPI/Issues/IssuesTestSteps.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  setAuthorizationHeaders(agent);
+  githubRepositoryName = repository;
+});
+
+When('a request is used to create an issue in the accounts repository', async function () {
+  this.response = await agent.post(`${urlBase}/repos/${this.username}/${githubRepositoryName}/issues`).send(query);
+});
+
+Then('the issue should have been created', function () {
+  expect(this.response.body.id).to.not.equal('undefined');
```
Comment (yesid-lopez) [inline test/features/support/GithubAPI/Issues/IssuesTestSteps.js]: That assertion is not recommended. (https://www.chaijs.com/api/bdd/#method_undefined)
May you prefer to validate that that property is a string or follows a specific structure

File: test/features/support/GithubAPI/Issues/IssuesTestSteps.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const urlBase = 'https://api.github.com';
+
+let githubRepositoryName;
+
+const query = { title: 'API Created Issue' };
+const updateQuery = { body: 'API Created Issue new body' };
+
+Given('a public repository of the account {string}', (repository) => {
+  setAuthorizationHeaders(agent);
```
Comment (yesid-lopez) [inline test/features/support/GithubAPI/Issues/IssuesTestSteps.js:18]: I'm seeing that you are setting this in all the scenarios, what about to create a step or move into a hook?

File: test/features/support/GithubAPI/Issues/LoggedUserInfoSteps.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+});
+
+Then('at least one public repository is expected', function () {
+  expect(this.response.body.public_repos).to.be.at.least(1);
+});
+
+When('a request is used to retrieve the accounts public repository information', async function () {
+  this.response = await agent.get(this.response.body.repos_url);
+
+  this.body = this.response.body.shift();
```
Comment (yesid-lopez) [inline test/features/support/GithubAPI/Issues/LoggedUserInfoSteps.js:23]: why do you need to use shift?

File: test/features/support/GithubAPI/Issues/LoggedUserInfoSteps.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+});
+
+Then('the repository must exist', function () {
+  expect(this.body.full_name).to.not.equal('undefined');
```
Comment (yesid-lopez) [inline test/features/support/GithubAPI/Issues/LoggedUserInfoSteps.js]: change this assertion

File: test/features/GithubApi.Issue.feature [hunk 5]
```diff
@@ -0,0 +1,20 @@
+Feature: Github Api Issues Create & Update Test
+    Scenario: Obtain and Verify the user logged in information
+        Given a Github account like 'manuelq12'
+        When a request is used to retrieve the account information
+        Then the response should contain a 'OK' status
+        And at least one public repository is expected
+        When a request is used to retrieve the accounts public repository information
+        Then the response should contain a 'OK' status
+        And the repository must exist
+
```
Comment (leonleo997) [inline test/features/GithubApi.Issue.feature:10]: delete empty line

**Review by yesid-lopez — DISMISSED** [review]

**Review by yesid-lopez — APPROVED** [review]

**Pull request merged**